new features added

luca-saggese opened the pull request.

luca-saggese committed:
added fake data and working soundcloud link

luca-saggese committed:
lot of improvements and fixes,
added playlist support, playback controls, share link, barcelona & rome
and more

luca-saggese committed:
added for error

luca-saggese committed:
minor fixes

luca-saggese committed:
more support for artist tracks

luca-saggese committed:
minor fixes and artwork display

luca-saggese committed:
minor fixes

luca-saggese committed:
playlist support

luca-saggese committed:
playlist fixes and random start

luca-saggese committed:
more places

luca-saggese committed:
fixes

luca-saggese committed:
fixes

luca-saggese committed:
added error info

trippedout commented:
Hey! Thanks for the pull request :) I hadn't any time to touch this project since it went live, so I'm glad someone finds it useful. 

If you could just amend your commit messages for something a bit more descriptive thatd make my job easier going through them. 

And my only real question is of the purpose SynoDLMSearch with cineblog and the copyright notice there - are there any licensing issues associated with that? Where/how is it pulling files?

luca-saggese commented:
I, i just commit the file search.php for mistake, i’ve removed after that.
your right the comments are a mess, but are quite all little fixes to css
and so on.
i’ve updated the three library and fixed some little problems with
linewidth being zero in same cases.
i’ve now made some improvements with also playlist support for choosing
songs.
don’t know if i can fix the comments, i’m not a git man and i find it
really confusing...

Anyway thanks for the great work, i really love it!

are there any improvement that could be done to speed up the rendering?

luca-saggese committed:
added facebook screenshot

luca-saggese committed:
allow generic stream (not only soundcoud)

luca-saggese committed:
fix for only one track not playing correctly

luca-saggese committed:
proxy implementation to avoid 403 error and cross domain errors

luca-saggese committed:
add delayed start

fixed incorrect elapsed time starting counting before song start

luca-saggese committed:
add proxy to avoid 403 for some ips

luca-saggese committed:
added preview for facebook

luca-saggese committed:
add show soundclouf form in case something goes wrong loading url

luca-saggese committed:
fix hand curson on playlist tracks

luca-saggese committed:
fix
